Source/Meadow.Windows.Samples/HMI/WinForms/WinForms.csproj

```diff
@@ -0,0 +1,21 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+ <PropertyGroup>
+	 <TargetFramework>net7.0-windows</TargetFramework>
+	 <UseWindowsForms>true</UseWindowsForms>
+	 <OutputType>Exe</OutputType>
+   <AssemblyName>App</AssemblyName>   
+ </PropertyGroup>
+ <PropertyGroup>
+   <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
+   <LangVersion>9.0</LangVersion>
+   <Nullable>enable</Nullable>
+ </PropertyGroup>
+ <ItemGroup>
+   <Compile Remove="MeadowApp.cs" />
+ </ItemGroup>
+ <ItemGroup>
+   <ProjectReference Include="..\..\..\..\..\Meadow.Core\source\implementations\windows\Meadow.Windows\Meadow.Windows.csproj" />
+   <ProjectReference Include="..\..\..\..\..\Meadow.Foundation\Source\Meadow.Foundation.Libraries_and_Frameworks\Graphics.MicroGraphics\Driver\Graphics.MicroGraphics.csproj" />
+   <ProjectReference Include="..\..\..\..\..\Meadow.Foundation\Source\Meadow.Foundation.Peripherals\Displays.WinForms\Driver\Displays.WinForms.csproj" />
```

Comment:
Do we want samples using direct references or should this be NuGet packages?

Reply:
Possibly ditto for some other csprojs in this PR.

Reply:
develop should be local refs

Reply:
and we should move some of these samples of out of Meadow.Core.Samples since they're using Foundation